README.md

```diff
@@ -1,9 +1,18 @@
-# nv
+<p align="center">
+  <a href="https://github.com/jcouture/nv">
+    <img src="https://i.imgur.com/Q9JoskI.png" alt="nv" />
```

Comment:
If you want to make sure this file is available _forever_, you can upload it in any GitHub comment area so it’s served by `user-images.githubusercontent.com` (you know, in case Imgur decides to stop serving it for free 🤷).

Reply:
Yep, done!